kong/runloop/balancer.lua

```diff
@@ -224,8 +224,18 @@ do
       elseif action == "removed" then
         local ok, err = healthchecker:remove_target(ip, port)
         if not ok then
-          log(ERR, "[healthchecks] failed adding a target: ", err)
+          log(ERR, "[healthchecks] failed removing a target: ", err)
+        end
+
+      elseif action == "health" then
+        local health_status
+        if address then
+          health_status = "HEALTHY"
+        else
+          health_status = "UNHEALTHY"
         end
+        log(WARN, "[healthchecks] healthchecker for ", healthchecker.name,
+            " reported health status changed to ", health_status)
```

Comment:
is `healthchecker.name` the same as the balancer name? I'd make the message more expclicit that it's the balancer that switches health state:

```lua
log(WARN, "[healthchecks] balancer ", healthchecker.name,
            " changed health status to ", health_status)
```